--- node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/xhr-utils.js.orig	2021-01-06 16:00:27.029643588 +0100
+++ node_modules/jest-environment-jsdom/node_modules/jsdom/lib/jsdom/living/xhr-utils.js	2021-01-06 16:06:04.662096891 +0100
@@ -93,7 +93,9 @@
   }
   const acahStr = response.headers["access-control-allow-headers"];
   const acah = new Set(acahStr ? acahStr.trim().toLowerCase().split(headerListSeparatorRegexp) : []);
-  const forbiddenHeaders = Object.keys(flag.requestHeaders).filter(header => {
+  const forbiddenHeaders = acah.has("*") ?
+  [] :
+  Object.keys(flag.requestHeaders).filter(header => {
     const lcHeader = header.toLowerCase();
     return !simpleHeaders.has(lcHeader) && !acah.has(lcHeader);
   });
